packages/@aws-cdk/cloudformation-include/test/nested-stacks.test.ts

```diff
@@ -0,0 +1,173 @@
+import '@aws-cdk/assert/jest';
+import * as s3 from '@aws-cdk/aws-s3';
+import * as core from '@aws-cdk/core';
+import * as path from 'path';
+import * as inc from '../lib';
+import * as futils from '../lib/file-utils';
+
+// tslint:disable:object-literal-key-quotes
+/* eslint-disable quotes */
+
+describe('CDK Include', () => {
+  let stack: core.Stack;
+
+  beforeEach(() => {
+    stack = new core.Stack();
+  });
+
+  test('can ingest a template with nested stacks and generate each nested template correctly', () => {
+    const parentTemplate = new inc.CfnInclude(stack, 'ParentStack', {
```

Comment:
Can you actually align the `app` and `stack` instances and their names in this case, so it matches the setup from the integ test? Since we're not checking the parent stack here, I feel like we have a blind spot here (for example, we don't check that the names of the asset parameters in the parent match the name of the asset parameters in the children, because of the different names).